node/impl/hierarchical/subnet.go

```diff
@@ -28,13 +28,14 @@ func (a *HierarchicalAPI) AddSubnet(
 	parent address.SubnetID, name string,
 	consensus uint64, minerStake abi.TokenAmount,
 	checkPeriod abi.ChainEpoch,
-	delegminer address.Address) (address.Address, error) {
-	return a.Sub.AddSubnet(ctx, wallet, parent, name, consensus, minerStake, checkPeriod, delegminer)
+	dm address.Address,
```

Comment:
Same as above, we should probably make a struct of opts to make the function signature more readable and dev-friendly.